.eslintrc.js

```diff
@@ -361,5 +361,12 @@ module.exports = {
         'import/no-unresolved': ['error', { ignore: ['^node:', 'vscode'] }],
       },
     },
+    {
+      files: ['packages/**'],
+      excludedFiles: ['packages/graphiql/**', 'packages/graphiql-react/**'],
```

Comment:
```suggestion
      // ignore React packages because it's ugly to have `async IIFE` inside `useEffect`
      excludedFiles: ['packages/graphiql/**', 'packages/graphiql-react/**'],
```